### PR TITLE
Changes to Ui concerning Boolean Tutorial

### LINKED ISSUE #129: [PE-D] Interactive Tutorial stuck in loop
Did not capitalize GATE by accident and ended up in an infinite loop regardless of what I typed. 

Can only leave with exit

EDIT: On reload, this problem does not require a problem to trigger and is always bugged out.

SECOND EDIT: On reading the documentation guide further, understood that GATE was meant to indicate type of gate I want. Still tagging this as a functionality error as this command is supposed to make me less confused.

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: brandonywl/ped#5

### LINKED ISSUE #135: [PE-D] Suggestions for tutorial boolean
For tutorial boolean, I think overvall is quite good. Here are some suggestions:
1. I don't quite understand 'add C GATE' when I saw it for the first time. I thought it asked me to enter 'add C GATE' and the program will ask me which GATE do you want to add. After I saw your UG I noticed that I should specify the GATE type, maybe you can change the instruction about this.
2. After I finished the tutorial, there is a congrats message. And I entered an invalid command it still printed the same message. I think may change it to something like "Opps! Please follow the format carefully."

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: ZhongNingmou/ped#1

### LINKED ISSUE #155: Implementation Issues
- [x] #123 
     - **`SetCircuitCommand`**: 
         - Update `SetCircuitCommand#toString()` to have voltage source also print template.
- [x] #129, #135, 
     - **`TutorialBooleanCommand`**: 
         - Also part of #154, update instructions on how to use `tutorial boolean`. 
             * `add C GATE` should be changed to an extremely specific example with no freedom of choice
- [x] #137, #142, #143, #144 
     - **`TutorialCircuitCommand`**: 
         * Check that the user is following the commands to the T, remove possibility for unavailable components to be added
         * Check that voltage cannot be set instead of a load component.
- [x] #136, #145 
     - **`AddCircuitCommand`**: 
         - Adding new components should print the old component. Make copy of old object.
- [x] #139 
     - **`Ui`**: 
         - Update `Ui#printGreeting()` with summary
- [x] #140
     - **`Ui`**: 
         - Update `Ui#LOGO` positioning
- [x] #146
     - **`BooleanTemplate`**: 
         - Prevent setting of unavailable indices in `BooleanTemplate`
- [x] #147
     - **ALL**: 
         - Customise Exceptions for every package.
- [x] #148, #149, #150 
     - **`AddCircuitCommand`**: 
         - Double check adding component that is not yet set. Can also change 0 values to "not set" when printing instead
- [x] #151, #152, #153 
     - **`Gate`**: 
         - Double check output of Xnor, Nor, Nand gates.

### PR DESCRIPTION
**Pull request by PraveenElango**
Changes to Ui concerning Boolean Tutorial including grammatical changes

### CODE REVIEW

File: unused/RemovedUi.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+//@@author PraveenElango
```
Comment (sevenseasofbri) [inline unused/RemovedUi.java:1]: What is the purpose of this file?

Comment (PraveenElango) [reply unused/RemovedUi.java:1]: Oh, I was browsing through the CS2113T site today and came across this - https://nus-cs2113-ay2021s1.github.io/website/admin/tp-w10.html#4-make-the-code-reposense-compatible. It is mentioned under the RepoSense section that we can include code individually written that is not used in the final product by placing it in the unused folder.

Comment (sevenseasofbri) [reply unused/RemovedUi.java:1]: Ohh  I see I see

Comment (sevenseasofbri) [reply unused/RemovedUi.java:1]: BTW did you test how the commands for boolean look with the jar file in a terminal
Longer lines are generally pushed to the next line thereby resulting in inconsistent tabs. So consider adding tabs and newline wherever necessary after checking. I made changes for circuit already (probably going to have merge conflict with your  current changes of circuit commands lol)

Comment (PraveenElango) [reply unused/RemovedUi.java:1]: I didn't test with jar file yet but I did add tabs and spaces when the sentences were too long for those in the Boolean tutorial. I'll test in the jar file and get back to you.

File: src/main/java/seedu/duke/ui/Ui.java [hunk 2]
```diff
@@ -41,21 +41,23 @@
         "Hooray! You're set to use the circuit builder of \n" + LOGO + "Enjoy! :)\n \ttype 'exit' to exit this mode."
             + "\n"};
 
-    public static final String[] BOOLEAN_INSTRUCTIONS = {"Lets get started! First chose a template you want to work "
+    public static final String[] BOOLEAN_INSTRUCTIONS = {"Let's get started! First chose a template you want to work "
```
Comment (hughjazzman) [inline src/main/java/seedu/duke/ui/Ui.java:44]: Maybe consider using `\t` similar to what @sevenseasofbri did for `CIRCUIT_INSTRUCTIONS`? To be consistent, `\n` can also be replaced with `System.lineSeparator()`.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -213,6 +213,7 @@ The second sequence diagram given below shows the detailed interaction that acce
 
 --------------------------------------------------------------------------------------------------------------------
 
+
```
Comment (hughjazzman) [inline docs/DeveloperGuide.md:216]: 😮 blank line

**Review by hughjazzman — COMMENTED** [review]
Is `TutorialBooleanCommand` not touched? I think would have to make it more strict i.e. `orderOfInstructions` needs to changed similar to how @sevenseasofbri did it for `TutorialCircuitCommand`.

As an example for this line
```
 Enter 'add C GATE' where GATE can be one of the following: and, or, nand, nor, xor, xnor.
```
If they used any `INPUT` other than `C` the app would tell the user to follow the instructions. Would be good for `INPUT.txt` to also test these.

File: src/main/java/seedu/duke/ui/Ui.java [hunk 4]
```diff
@@ -20,17 +20,17 @@
             + "\t    | | |_| | || (_) | |  | | (_| | | | |  | | (_) | (_| |  __/_|\n"
             + "\t    |_|\\__,_|\\__\\___/|_|  |_|\\__,_|_| |_|  |_|\\___/ \\__,_|\\___(_)\n";
 
-    public static final String[] CIRCUIT_INSTRUCTIONS = {"Lets get started! First we will choose a template to work "
-            + "with. In the app you can choose from rc, rl, lc and r templates." + System.lineSeparator() + "\tHowever,"
-            + " for this tutorial, we will use a Resistor-Capacitor template." + System.lineSeparator()
+    public static final String[] CIRCUIT_INSTRUCTIONS = {"Let's get started! First we will choose a template to work "
+            + "with. In the app you can choose from rc, rl, lc, and r templates." + System.lineSeparator() + "\tHowever"
```
Comment (hughjazzman) [inline src/main/java/seedu/duke/ui/Ui.java:24]: Looking at #162 maybe a newline could be added before "In the app"?

Comment (sevenseasofbri) [reply src/main/java/seedu/duke/ui/Ui.java:24]: Made this change yest in my system on a branch i'm currently working on for the PPP

**Review by hughjazzman — APPROVED** [review]
LGTM

**Pull request merged**